Pythonwin/pywin/framework/editor/__init__.py

```diff
@@ -100,4 +100,4 @@ def SetEditorFontOption(option, newValue):
     SetEditorOption(option, str(newValue))
 
 
-from pywin.framework.editor.color.coloreditor import editorTemplate
+import pywin.framework.editor.color.coloreditor  # nopycln: import # Adds doc template
```

Comment:
Assuming this side-effect is wanted here: https://github.com/mhammond/pywin32/blob/377a3efb3fd01602555066150e3c76e74fa5aa4f/Pythonwin/pywin/framework/editor/color/coloreditor.py#L639-L646